backoffice/templates/backoffice/certificate.html

```diff
@@ -93,7 +93,7 @@ <h2>{{ course.display_name_with_default }}</h2>
                   <td>{% if instructor_task.task_type == 'certificate-generation' %}{% trans "Non Verified" %}{% else %}{% trans "Verified" %}{% endif %}</td>
                   <td class="task-certificate-pdf">
                     {% if instructor_task.task_type == 'certificate-generation' %}
-                        <a href="{{certificate_base_url}}{{instructor_task.task_output.test_certificate_filename}}">{% trans "Sample" %}</a>
+                        <a href="/media/{{certificates_directory}}/{{instructor_task.task_output.test_certificate_filename}}">{% trans "Sample" %}</a>
```

Comment:
Why not using the [`MEDIA_URL`](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-MEDIA_URL) setting instead?

```suggestion
                        <a href="{{ MEDIA_URL }}/{{certificates_directory}}/{{instructor_task.task_output.test_certificate_filename}}">{% trans "Sample" %}</a>
```

Reply:
You can't do this because it is a Mako template. It can be done with specific imports but since there are already many `/media` mentions in the code base, we should leave it up to another PR.